New changes SQL/XT Voidzone add.sql

```diff
@@ -0,0 +1 @@
+UPDATE creature_template SET ScriptName ='npc_xt_void_zone' WHERE entry = 34001;
```

Comment:
Delete the whole update file and just copy the updated sql script into a new one?